src/main/kotlin/command/Timer.kt

```diff
@@ -0,0 +1,84 @@
+package command
+
+import chat.ChatUtility
+import chat.Formatting
+import io.papermc.paper.command.brigadier.CommandSourceStack
+import net.kyori.adventure.bossbar.BossBar
+import net.kyori.adventure.text.Component
+import org.bukkit.Bukkit
+import org.bukkit.entity.Player
+import org.bukkit.scheduler.BukkitRunnable
+import org.incendo.cloud.annotations.Argument
+import org.incendo.cloud.annotations.Command
+import org.incendo.cloud.annotations.Default
+import org.incendo.cloud.annotations.Permission
+import org.incendo.cloud.annotations.processing.CommandContainer
+import plugin
+import util.Sounds.PLING
+import util.timeRemainingFormatted
+
+@Suppress("unused", "unstableApiUsage")
+@CommandContainer
+class Timer {
+    @Command("timer start <time> [countdown]")
+    @Permission("tbd.command.timer")
+    fun timerStart(css: CommandSourceStack, @Argument("time") time: Int, @Argument("countdown") @Default("-1") countdown: Int) {
+        if(time in 1..60) {
+            if(css.sender is Player) {
+                val player = css.sender as Player
+                ChatUtility.broadcastDev("Timer <dark_gray>(${time.timeRemainingFormatted()})</dark_gray> started by ${player.name}.", false)
+                TimerFunction.startTimer(time, countdown)
+            }
+        }
+    }
+}
+
+object TimerFunction {
+    fun startTimer(timeInMinutes: Int, countdown: Int) {
```

Comment:
Why the separate object for this rather than just putting the function in the class?